.travis/linux.win.download.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+CACHE_DIR=$TRAVIS_BUILD_DIR/apt_mingw_cache/$1
+mkdir -p $CACHE_DIR
+
+pushd $CACHE_DIR
+
+apt-get --print-uris --yes install $MINGW_PACKAGES | grep ^\' | cut -d\' -f2 > downloads.list
+wget -N --input-file downloads.list
```

Comment:
> wget does not download dependencies either.

But `apt-get --print-uris install <packages>` prints all URIs that need to be downloaded.

> After downloading a package, it should be checked against Packages.

I expect `apt-get` to verify the integrity of packages in the cache prior to installing them. Are you sure it doesn't that?